transient/transient.py

```diff
@@ -30,18 +31,33 @@ def __format_new_qemu_args(self) -> List[str]:
             # Use userspace networking (so no root is needed), and bind
             # localhost port 5555 to guest port 22
             # TODO: try to find an available port
-            new_args.extend(["-net", "nic,model=e1000",
-                             "-net", "user,hostfwd=tcp::5555-:22"])
+            new_args.extend([
+                "-netdev",
+                "user,id=sshdev0,hostfwd=tcp::{}-:22".format(self.config["ssh_port"]),
+                "-device",
+                "virtio-net-pci,netdev=sshdev0"
```

Comment:
what is the reason for the change here? does this not now require the guest have virtio support?

Reply:
`virtio` isn't needed, but `-net` is deprecated in favor of `-netdev`

Reply:
Gotcha. It might be good to leave the older approach if it means we're compatible with more stuff. I guess we should pick a minimum qemu version. Do you have a recommendation? Maybe whatever is in the Ubuntu 18.04 repos?

Reply:
AFAIK ubuntu 18.04 is on 2.11 and `-netdev` was introduced before that.